source/solvers/navier_stokes_base.cc

```diff
@@ -74,6 +74,7 @@ NavierStokesBase<dim, VectorType, DofsType>::NavierStokesBase(
   , velocity_fem_degree(p_nsparam.fem_parameters.velocity_order)
   , pressure_fem_degree(p_nsparam.fem_parameters.pressure_order)
   , number_quadrature_points(p_nsparam.fem_parameters.velocity_order + 1)
+  , pressure_scaling_factor(p_nsparam.stabilization.pressure_scaling_factor)
```

Comment:
Why do you need to store this a s a member of the class? I mean the parameter is always available no?